Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+# add our user and group first to make sure their IDs get assigned consistently
+RUN groupadd -r sentry && useradd -r -m -g sentry sentry
+
+RUN yum update -y && yum clean all
+
+RUN yum install -y \
+        gcc \
+        python2-devel \
+        python2-pip \
+        redhat-rpm-config \
+    && yum clean all
+
+# Sane defaults for pip
+ENV PIP_NO_CACHE_DIR off
+ENV PIP_DISABLE_PIP_VERSION_CHECK on
+
+# grab gosu for easy step-down from root
+ENV GOSU_VERSION 1.11
+RUN set -x \
+    && yum install -y wget && yum clean all \
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$(if [ `arch` = 'x86_64' ]; then echo 'amd64'; else echo `arch`; fi)" \
```

Comment:
this is super interesting, it's the first time I'm seeing it, can you elaborate at some point on gosu?

Reply:
Link to gosu: https://github.com/tianon/gosu

It's used in the sentry container entrypoint script, which we are also using.  It's basically a an su/sudo tool with the tty handling issues.